src/app/shared/common-shared.module.ts

```diff
@@ -0,0 +1,44 @@
+import { NgModule } from '@angular/core';
```

Comment:
But can we change the name of file to just `shared.module.ts` or change the folder name to `common` and then file name to `common.module.ts`.